### DISCUSSION TITLE
Offers

### DISCUSSION THREAD

**Discussion by Tetsunokishi** [eritislami/evobot#243] [Ideas]
Will there be commands for moderation? (Ban (unban), warm (unwarn), kick, mute (unmute))
I would really like to see these functions in the bot

**Comment by imZesk** [comment]
Is a musicbot, if u want add those functions is pretty easy.

**Comment by dylanjamesdev** [comment]
^

**Comment by paraswtf** [comment]
I don't think that he'll add those features, coz it's just a music bot currently
